docs/api-reference/horizon-ai/aggregation/query-dsl/group-by-clause.mdx

```diff
@@ -128,3 +128,84 @@ va.select(...).group_by(
 ```
 </CodeGroup>
 
+## Group by any attribute from the table expression
```

Comment:
Maybe have separate sections for grouping by SourceColumnExpr and grouping by any data columns

Also _columns_, not attributes.